serverless/install/installing-knative-serving.adoc

```diff
@@ -24,4 +16,4 @@ include::modules/knative-serving-advanced-config.adoc[leveloffset=+1]
 [id="next-steps_installing-knative-serving"]
 == Next steps
 
-* For cloud events functionality on {ServerlessProductName}, you can install the Knative Eventing component. See the documentation on xref:../../serverless/install/installing-knative-eventing.adoc#installing-knative-eventing[Installing Knative Eventing].
+* If you want to use Knative event-driven architecture you can xref:../../serverless/install/installing-knative-eventing.adoc#installing-knative-eventing[install Knative Eventing].
```

Comment:
```suggestion
* If you want to use Knative event-driven architecture you can xref:../../serverless/install/installing-knative-eventing.adoc#installing-knative-eventing[Install Knative Eventing].
```

Reply:
This isn't the literal name of the topic, so I'm not sure the capital letter makes sense here - I was hoping to have the xref just read like part of the sentence